treeherder/config/settings.py

```diff
@@ -39,6 +39,7 @@
 if env("HEROKU_REVIEW_APP", default=False):
     # This is to support Heroku Review apps which host is different for each PR
     SITE_URL = "https://{}.herokuapp.com".format(env("HEROKU_APP_NAME"))
+    PULSE_AUTO_DELETE_QUEUES = True
```

Comment:
Get this out of this if condition and do this instead:
`PULSE_AUTO_DELETE_QUEUES = env("PULSE_AUTO_DELETE_QUEUES", default=False))`

The variable will be defined because of `app.json`

Could you also add the variable to docker-compose? Set it to true.

Reply:
Oh I thought that what goes into the app.json is for production.
And heroku review app has a different case as I defined inside the if condition there
Just making the changes
I just got the real situation
Thanks for reviews

Reply:
But one this is still not clear, how is this going to make it true in production
Can you elaborate on this a bit please

Reply:
Understood this one too.
Thanks @armenzg